source/retryable-reads/tests/estimatedDocumentCount.yml

```diff
@@ -1,9 +1,11 @@
 runOn:
     -
         minServerVersion: "4.0"
+        maxServerVersion: "4.9.0"
```

Comment:
Same note about `maxServerVersion` being inclusive. This should probably be "4.8.99".